GiniVision/Classes/Networking/DocumentsService.swift

```diff
@@ -17,6 +17,7 @@ final class DocumentsService: DocumentServiceProtocol {
     
     init(sdk: GiniSDK) {
         self.giniSDK = sdk
+        self.giniSDK.sessionManager.logIn()
```

Comment:
I don't want to be this guy, but do you think the constructor is a suitable place for something like a login?
In my opinion, a construction should only init the object and not start such tasks/side effects.
It might be inconvenient for users.